packages/react-environment/src/api-providers/web3/EvmChainMixersInfo.ts

```diff
@@ -0,0 +1,93 @@
+import { MixerTitle } from '@webb-dapp/react-environment';
+import { Storage } from '@webb-dapp/utils';
+import { MixerStorage, evmChainStorageFactory } from '../../../../apps/src/configs/storages/EvmChainStorage';
```

Comment:
When attempting to import from `@webb-dapp/apps/configs/storages/EvmChainStorage`, the file cannot be found.

Reply:
Hmm, @AhmedKorim any idea here?